src/server/api/controllers/profile.controller.js

```diff
@@ -37,7 +38,63 @@ const getAdminsProfile = async (userId) => {
     return error.message;
   }
 };
+
+const editMentorProfile = async (mentorId, updatedMentor) => {
+  // validate that user exists and return 404 if it doesn't
+  const getUser = await knex('users').select('id').where('id', mentorId);
+  if (getUser[0].id !== mentorId) return 404;
+
+  // validate that name is not an empty string and return 400 if it is
+  if (updatedMentor.name === null) return 400;
```

Comment:
You could write:
```suggestion
  if (!updatedMentor.name) return 400;
```
This covers `null`, `undefined` and `''`